android-app-examples/SNetDemo/app/src/main/java/com/example/snetdemo/StyleTransferService.java

```diff
@@ -0,0 +1,33 @@
+package com.example.snetdemo;
+
+import java.io.Closeable;
+import java.math.BigInteger;
+
+import io.singularitynet.sdk.client.FixedPaymentChannelPaymentStrategy;
+import io.singularitynet.sdk.client.PaymentStrategy;
+import io.singularitynet.sdk.client.ServiceClient;
+import io.singularitynet.service.styletransfer.StyleTransferGrpc;
+
+public class StyleTransferService implements Closeable
```

Comment:
Same as for semantic segmentation. I would suggest move this code into `StyleTransferActivity` `onCreate()` and `onDestroy()` methods.

Reply:
Done